src/SamplesApp/UITests.Shared/Windows_UI_Notifications/BadgeNotificationTests.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Data.Xml.Dom;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace UITests.Windows_UI_Notifications
+{
+	[SampleControlInfo("Windows.UI.Notifications")]
+	public sealed partial class BadgeNotificationTests : Page
+    {
+        public BadgeNotificationTests()
+        {
+            this.InitializeComponent();
+        }
+
+		private void SetBadge_Click(object sender, RoutedEventArgs e)
+		{
+			var badgeXml = BadgeUpdateManager.GetTemplateContent(BadgeTemplateType.BadgeNumber);
+			var badgeGlyphXml = BadgeUpdateManager.GetTemplateContent(BadgeTemplateType.BadgeGlyph);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'badgeGlyphXml'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5496594)